src/Symfony/Component/Security/Http/RateLimiter/DefaultLoginRateLimiter.php

```diff
@@ -28,11 +28,19 @@ final class DefaultLoginRateLimiter extends AbstractRequestRateLimiter
 {
     private RateLimiterFactory $globalFactory;
     private RateLimiterFactory $localFactory;
+    private ?string $secret;
```

Comment:
sorry I missed this one:
```suggestion
    private string $secret;
```